### DISCUSSION TITLE
edgeComputing/comparison

### DISCUSSION THREAD

**Discussion by giscus** [LaoYutang/blogGiscus#9] [Announcements]
# edgeComputing/comparison

本文将横向对比网心云、猕猴桃、甜糖三款带宽分享服务，2023年5月实际测试记录。

https://blog.laoyutang.cn/edgeComputing/comparison.html

**Comment by WalkingDead-Chen** [comment]
可以去试下“点心云”，没有Docker版本就是，有一段时间说要出Docker版本，后面又没动静了

**Comment by 1418615553** [comment]
赞 说的好详细
